reconcile/templating/renderer.py

```diff
@@ -331,9 +345,8 @@ def reconcile_template_collection(
                     persistence_transaction.write(output)
 
     def reconcile(
-        self, persistence: FilePersistence, ruamel_instance: yaml.YAML
+        self, gql_api: GqlApi, persistence: FilePersistence, ruamel_instance: yaml.YAML
     ) -> None:
-        gql_api = init_from_config(validate_schemas=False)
```

Comment:
initializing gql_api is moved to run and passed into reconcile in order to take advantage of the url and token attributes retrieved during [gql init](https://github.com/app-sre/qontract-reconcile/blob/f538b381ae60a5bb6ebbb9bdb3f0305146e43fdf/reconcile/utils/gql.py#L392)

Reply:
can you check if we really need to call `init_from_config` here instead of using already inited `gql_api`? And we should be able to get `commit` field directly from that `gql_api` object.

Reply:
Thank you for the call outs. Code has been simplified.